docusaurus.config.ts

```diff
@@ -0,0 +1,141 @@
+import { themes as prismThemes } from "prism-react-renderer";
+import type { Config } from "@docusaurus/types";
+import type * as Preset from "@docusaurus/preset-classic";
+
+const config: Config = {
+  title: "Mercurius",
+  tagline: "Mercurius is a GraphQL adapter for Fastify",
+  favicon: "favicon.ico",
+
+  // Set the production url of your site here
+  url: "https://your-docusaurus-site.example.com",
+  // Set the /<baseUrl>/ pathname under which your site is served
+  // For GitHub pages deployment, it is often '/<projectName>/'
+  baseUrl: "/",
+
+  // GitHub pages deployment config.
+  // If you aren't using GitHub pages, you don't need these.
+  organizationName: "facebook", // Usually your GitHub org/user name.
+  projectName: "docusaurus", // Usually your repo name.
+
+  onBrokenLinks: "throw",
+  onBrokenMarkdownLinks: "warn",
+
+  // Even if you don't use internationalization, you can use this field to set
+  // useful metadata like html lang. For example, if your site is Chinese, you
+  // may want to replace "en" with "zh-Hans".
+  i18n: {
+    defaultLocale: "en",
+    locales: ["en"]
+  },
+
+  presets: [
+    [
+      "classic",
+      {
+        docs: {
+          sidebarPath: "./sidebars.ts",
+          // Please change this to your repo.
+          // Remove this to remove the "edit this page" links.
+          editUrl:
+            "https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/"
+        },
+        blog: {
+          showReadingTime: true,
+          feedOptions: {
+            type: ["rss", "atom"],
+            xslt: true
+          },
+          // Please change this to your repo.
+          // Remove this to remove the "edit this page" links.
+          editUrl:
+            "https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/",
+          // Useful options to enforce blogging best practices
+          onInlineTags: "warn",
+          onInlineAuthors: "warn",
+          onUntruncatedBlogPosts: "warn"
+        },
+        theme: {
+          customCss: "./src/css/custom.css"
+        }
+      } satisfies Preset.Options
+    ]
+  ],
+  plugins: [
+    [
+      require.resolve("docusaurus-lunr-search"),
+      {
+        // Optional: Specify options here
+        languages: ["en"] // Array of languages, default is ['en']
+      }
+    ]
+  ],
+  themeConfig: {
+    // Replace with your project's social card
+    image: "img/docusaurus-social-card.jpg",
```

Comment:
Is this surfaced anywhere? Probs worth changing this?

Reply:
I think we can remove it

Reply:
I removed it , see PR here: https://github.com/mercurius-js/website/pull/1/commits/54d8a2dc7ab12c4165ec309fd51e6865fc432520